source/catalog.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body>
+  <header class="main-header">
+    <a class="main-header__logo">
+      <h1 class="visually-hidden">Сайт Седоны.</h1>
+      <img src="/images/logo-sedona.svg" width="185" height="96" alt="Логотип Седоны.">
```

Comment:
Неверно указан путь к файлу изображения:
```suggestion
      <img src="images/logo-sedona.svg" width="185" height="96" alt="Логотип Седоны.">
```
Браузер не найдёт изображение:
![image](https://user-images.githubusercontent.com/17790771/183128118-0ff69000-0c4a-493d-be5c-1c5761e56692.png)
Нужно исправить во всех `img`.